src/Halogen/Svg/Attributes.purs

```diff
@@ -413,9 +451,27 @@ d = attr (AttrName "d") <<< joinWith " " <<< unwrapNewtype
 text_anchor :: forall r i . TextAnchor -> IProp (text_anchor :: String | r) i
 text_anchor = attr (AttrName "text-anchor") <<< printTextAnchor
 
-font_size :: forall r i. FontSize -> IProp (font_size :: String | r) i
+font_family :: forall r i. String -> IProp (fontFamily :: String | r) i
+font_family = attr (AttrName "font-family")
+
+font_size :: forall r i. FontSize -> IProp (fontSize :: String | r) i
```

Comment:
I think this counts as a breaking change because the label was previously `font_size` but will now be `fontSize`.

Reply:
I've reverted this in the latest commit. I will open an issue to try to conform all the naming styles.

Reply:
However. I should note that there are still breaking changes in this PR. For example, you can no longer set the `fill` attribute of a `line`. This is intentional, because although it would be valid SVG, changing the `fill` of a `line` has no effect. But it is a breaking change.

Reply:
And this PR now fixes #3.

Reply:
Personally, I'd rather use camelCase for the names. We can fix this in a separate PR, but it'll make the diff cleaner. Looks like the next release will be breaking anyways, so no worries about that.